src/west/commands/command.py

```diff
@@ -84,9 +98,15 @@ def run(self, args, unknown):
         :param unknown: unknown arguments present on the command line;
                         this must be empty if the constructor
                         was passed accepts_unknown_args=False.
+        :param topdir: top level directory of west installation, or None;
+                       this is stored as an attribute before do_run() is
+                       called.
         '''
         if unknown and not self._accept_unknown:
             self.parser.error('unexpected arguments: {}'.format(unknown))
+        if not topdir and self.requires_installation:
+            sys.exit(_NO_TOPDIR_MSG_FMT.format(os.getcwd()))
```

Comment:
Why not `log.die()` here?

Reply:
I'm preserving the existing behavior, which doesn't use colorama etc. Happy to change it to use log.die if you want.

Reply:
Yes please, let's be consistent across the board